www/hodl01/semestralka/js/main.js

```diff
@@ -0,0 +1,50 @@
+$(function () {
+    //Instagram API
+    var ptoken = '1404870044.fa0d083.290093ca2a4d4a7f90e28bf31ec3570d',
```

Comment:
token ziskat z autentizace (prihlaseni) uzivatele, tj je potreba naimplementovat autorization flow instagramu

Reply:
to stejne udelej navic, jinak by tva prace nemohla byt dostatecne kvalifikovana jako SP, 
- treba to udelej tak, ze defaultne se zobrazi tvuj instagram, a bude tam moznost pro prihlaseni uzivatele, hlavne to budu hodnotit